index.html

```diff
@@ -4,9 +4,52 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
     <title>Weather Report</title>
+    <link href="styles/index.css" rel="stylesheet" />
 </head>
 <body>
+    <section>
+        <h1 id="titleCityName">Weather Report for Bridgewater</h1>
+    </section>
+
+    <section class="main-container">
+
+        <section>
+            <h2>Temperature</h2>
+            <button id="increaseTemp" class="big-text">Click to increase</button>
+            <h1 id="tempDisplay"><p class="text-color-blue">Current Temperature: 80 F</p></h1>
```

Comment:
We don't nest HTML into `h1` except maybe `span` tags, so let's get rid of the `p` tag altogether:

```suggestion
            <h1>Current Temperature: <span id="tempDisplay">80</span> F</h1>
```

By moving the id to surround _only_ the temperature value, we don't have to worry about extra text or anything being overwritten in JavaScript